security/access_control.rst

```diff
@@ -57,7 +57,12 @@ Take the following ``access_control`` entries as an example:
                 <rule path="^/admin" role="ROLE_USER_IP" ip="127.0.0.1" />
                 <rule path="^/admin" role="ROLE_USER_HOST" host="symfony\.com$" />
                 <rule path="^/admin" role="ROLE_USER_METHOD" methods="POST, PUT" />
-                <rule path="^/admin" role="ROLE_USER" />
+                <rule path="^/admin" roles="ROLE_ADMIN, ROLE_MANAGER" />
+                <!-- same as: -->
+                <rule path="^/admin">
+                    <role>ROLE_MANAGER</role>
+                    <role>ROLE_ADMIN</role>
+                </rule>
```

Comment:
Any good reason to even show this one? Does it possibly offer anything better than the above? I'd rather show just *one* way

Reply:
Fixed.